pages/release-notes.mdx

```diff
@@ -7,14 +7,17 @@ import { Steps } from "nextra/components"
 
 <Steps>
 <Step>
-## November 2nd 2023 - v0.73.0
+## 2 November 2023 - v0.73.0
```

Comment:
I wasn't sure that "2 November" is proper English. 😂

Reply:
It is! Let's keep the dates consistent.